drools-ansible-rulebook-integration-api/src/main/java/org/drools/ansible/rulebook/integration/api/rulesmodel/RulesModelUtil.java

```diff
@@ -13,50 +13,31 @@
 
 public class RulesModelUtil {
 
+    @Deprecated // no longer to be used
     public static final String ORIGINAL_MAP_FIELD = "$_ORIGINAL_$_MAP_$_FIELD_$";
```

Comment:
I would totally remove this instead of deprecating. I don't see any reason to keep it around and I'm afraid it could just be confusing.

Reply:
removed with https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/56/commits/3944ca7ae170fcfb180603f6917770d2d6127dab

Reply:
(oops)

Reply:
removed it with https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/56/commits/b536459df5a73ce8fd20cab9415e272df07f6f1f